src/lib/y2security/security_policies/disa_stig_policy.rb

```diff
@@ -0,0 +1,207 @@
+# Copyright (c) [2022] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2security/security_policies/policy"
+require "y2security/security_policies/action"
+require "y2security/security_policies/scopes"
+require "y2security/security_policies/issue"
+require "y2network/startmode"
+require "y2network/connection_config/wireless"
+require "bootloader/grub2base"
+
+module Y2Security
+  module SecurityPolicies
+    # DISA STIG Security Policy.
+    # DISA = US Defense Information Systems Agency
+    # STIG = Security Technical Implementation Guides
+    class DisaStigPolicy < Policy
+      include Yast::I18n
+
+      # @see Policy
+      def initialize
+        textdomain "security"
+
+        # TRANSLATORS: This is a security policy name.
+        #   "Defense Information Systems Agency" is from the USA, https://disa.mil/
+        #   STIG = Security Technical Implementation Guides
+        super(:disa_stig, _("Defense Information Systems Agency STIG"), ["scap-security-guide"])
+      end
+
+    private
+
+      # @see Policy
+      def issues_for(scope)
+        case scope
+        when Scopes::Network
+          network_issues(scope)
+        when Scopes::Storage
+          storage_issues(scope)
+        when Scopes::Firewall
+          firewall_issues(scope)
+        when Scopes::Bootloader
+          bootloader_issues(scope)
+        else
+          []
+        end
+      end
+
+      # Returns the issues in the network configuration
+      #
+      # * Wireless devices are not supported
+      #
+      # @param scope [Scopes::Network]
+      # @return [Array<Issue>]
+      def network_issues(scope)
+        config = scope.config
+        conns = find_wireless_connections(config)
+        return [] if conns.empty?
```

Comment:
NP: IMHO this is a useless optimization, `each_with_object([])` below will do just fine

```suggestion
```